lib/cb/utils/api.rb

```diff
@@ -46,9 +46,10 @@ def cb_delete(path, options={}, &block)
 
       def execute_http_request(http_method, uri, path, options={}, &block)
         self.class.base_uri(uri || Cb.configuration.base_uri)
-        cb_event(:"cb_#{http_method.to_s}_before", path, options, nil, &block)
+        start_time = Time.now.to_f
+        cb_event(:"cb_#{ http_method }_before", path, options, nil, start_time - start_time, &block)
```

Comment:
Yeah, was thinking that as well as I typed that response out. One sec, will do real quick.